### ISSUE TITLE
New files are hidden so often missed from commits

### ISSUE THREAD

**Issue by garth** [sublimehq/sublime_merge#1424]
**Version info**

 - OS: Macos
 - Build: 2068

**Description**

New files are hidden by default when making commits, so they often missed.

**Steps to reproduce**

1. add a new file together with other changes
2. open smerge and only the changed files are visible

**Expected behavior**

Files should never be hidden

**Comment by michaellenaghan** [comment]
Just chiming in to agree! (It's my first day of using Sublime Merge, and that struck me as odd right away.)

**Comment by dmaz-np** [comment]
Yeah, sublime merge does make it too easy to overlook these files and that is a bit frustrating.
